ext/shim/extconf.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+require "mkmf"
+
+# Makes all symbols private by default to avoid unintended conflict
+# with other gems. To explicitly export symbols you can use RUBY_FUNC_EXPORTED
+# selectively, or entirely remove this flag.
+append_cflags("-fvisibility=hidden")
+
+create_makefile("shim/shim")
```

Comment:
Should it be `systemd/journal/shim`?

```diff
diff --git a/ext/shim/extconf.rb b/ext/shim/extconf.rb
index 94abd76..a53b749 100644
--- a/ext/shim/extconf.rb
+++ b/ext/shim/extconf.rb
@@ -7,4 +7,4 @@ require "mkmf"
 # selectively, or entirely remove this flag.
 append_cflags("-fvisibility=hidden")

-create_makefile("shim/shim")
+create_makefile("systemd/journal/shim")
```

it seems that it can't load by `require "systemd/journal/shim"`.
(`require 'shimshim' is okay, but it is not intended I guess)

Reply:
you're correct, i used `bundle new shim --ext` to generate the skeleton for the native code and missed this spot when updating :)